docs/0001-digest-algorithms.md

```diff
@@ -0,0 +1,22 @@
+# OCFL Community Extension 0001: Digest Algorithms
+
+  * Authors: OCFL Editors
+  * OCFL Version: 1.0
+  * Obsoletes: n/a
+  * Obsoleted by: n/a
+
+## Overview
+
+This extension is an index of additional digest algorithms that are defined in community extensions. It provides a list of digest algorithm names that may be used to indicate the given algorith in `fixity` blocks of OCFL Objects, and links their defining extensions.
+
+## Digest Algorithms Defined in Community Extensions
+
+------------------------------------------------------
+| Digest Algorithm Name | Community Extension | Note |
+------------------------------------------------------
+| ... none yet ...      |                     |      |
+------------------------------------------------------
+
+## Maintenance
+
+In order to have an additional digest algorithm listed here, please submit an extension describing it and with a name that does not conflict with those defined in the OCFL Specification or in other community extensions, and is preferably in common use for the given algorithm.
```

Comment:
We should ask them to update this extension, not submit a new one.

Reply:
Will do :+1: